packages/core/src/lib/auth/auth.controller.ts

```
@@ -149,6 +151,43 @@ export class AuthController {
+
+			const url = new URL(`${process.env.API_BASE_URL ?? 'http://localhost:3000'}/api/integration/zapier/oauth/callback`);
+			if (zapier_state) {
+				url.searchParams.append('state', zapier_state);
+			}
+			if (zapier_redirect_uri) {
+				url.searchParams.append('zapier_redirect_uri', zapier_redirect_uri);
+			}
+			return res.redirect(url.toString());
+
+			// Regular login flow
+			return res.redirect('/dashboard');
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)